framework_tvb/tvb/core/neocom/h5.py

```diff
@@ -139,6 +143,22 @@ def store_complete(datatype, base_dir, generic_attributes=GenericAttributes()):
     return index_inst
 
 
+def store_complete_to_dir(datatype, base_dir, generic_attributes=GenericAttributes()):
+    h5_class = REGISTRY.get_h5file_for_datatype(datatype.__class__)
+    storage_path = path_by_dir(base_dir, h5_class, datatype.gid)
+
+    index_inst = store_complete(datatype, storage_path, h5_class, generic_attributes)
+    return index_inst
+
+
+def store_complete_to_op_dir(datatype, op_id, project_name, generic_attributes=GenericAttributes()):
```

Comment:
Could this remain called `store_complete` and the common `store_complete` function at line 128 be private?

Reply:
Of course